### PR TITLE
69 when skipping species classification and running staramr pipeline fails

### PR DESCRIPTION
**Pull request by mattheww95**
Updated StarAMR point finder database selection process to address error of no top species hit.

### CODE REVIEW

**Review by apetkau — CHANGES_REQUESTED** [review]
This looks good. A few comments.

File: subworkflows/local/annotate_genomes.nf [hunk 1]
```diff
@@ -63,15 +62,25 @@ workflow ANNOTATE_GENOMES {
...
+
         def db_star = [] // set default value for database
         if(params.staramr.db){
             db_star = Channel.value("${params.staramr.db}")
         }
+
         // Dump db versions
-        STARAMR_DUMP_DB_VERSIONS(db_star)
+        // Removed as data is in results
+        //STARAMR_DUMP_DB_VERSIONS(db_star)
```
[the remark this answers is not in the document]
Comment (apetkau) [reply subworkflows/local/annotate_genomes.nf:73]: Also, could you delete the code under the `modules/local` directory for this module?

File: nextflow.config [hunk 2]
```diff
@@ -133,26 +133,6 @@ params {
...
-    lx_min_aa_match_cov = 80.0
-    lx_max_target_seqs = 10
-    lx_extraction_mode = "raw"
-    lx_report_mode = "normal"
-    lx_report_prop = "locus_name"
-    lx_report_max_ambig = 0
-    lx_report_max_stop = 0
-
-    // Overide an allele calling scheme, this will be applied globally if auto selection is not opted for
-    allele_scheme = null
```
[the remark this answers is not in the document]
Comment (mattheww95) [reply nextflow.config:154]: It appears that when I used rebase, I did not properly resolve all merge conflicts and the parameters were repeated. The still exist in the files here: https://github.com/phac-nml/mikrokondo/blob/c91a887f03c5373c8749873c8a2ae8245a8c2f60/nextflow.config#L116 I just removed the duplicates.

File: subworkflows/local/annotate_genomes.nf (hunk 1 above)
[the remark this answers is not in the document]
Comment (mattheww95) [reply subworkflows/local/annotate_genomes.nf]: fixed in: 4ec2383

File: tests/pipelines/main.from_assemblies.nf.test [hunk 3]
```diff
@@ -163,6 +163,73 @@ nextflow_pipeline {
...
+            assert iridanext_samples.ecoli_GCA_000947975.findAll { it.path == "Assembly/Quality/QUAST/ecoli_GCA_000947975/ecoli_GCA_000947975.transposed_ecoli_GCA_000947975.quast.quality.tsv" }.size() == 1
+            assert iridanext_samples.ecoli_GCA_000947975.findAll { it.path == "Assembly/Quality/SeqKitStats/ecoli_GCA_000947975.seqkit.stats.summary.tsv" }.size() == 1
+            assert iridanext_samples.ecoli_GCA_000947975.findAll { it.path == "FinalReports/FlattenedReports/ecoli_GCA_000947975.flat_sample.json.gz" }.size() == 1
+
+            // output metadata
+            def ecoli_metadata = iridanext_metadata.ecoli_GCA_000947975
+            assert ecoli_metadata."QC Status" == "FAILED"
+            assert ecoli_metadata."n50 Value" == 122025
+            assert ecoli_metadata."Length Value" == 5299656
+            assert ecoli_metadata."nr contigs Value" == 123
```
[the remark this answers is not in the document]
Comment (mattheww95) [reply tests/pipelines/main.from_assemblies.nf.test:225]: fixed in: 4ec2383

Just waiting to confirm CI tests run

File: nextflow.config (hunk 2 above)
[the remark this answers is not in the document]
Comment (apetkau) [reply nextflow.config:154]: Oh, okay. Thanks.

**Review by apetkau — APPROVED** [review]
Everything looks great now. Thanks.

**Pull request merged**